spec/support/factory_bot.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+end
+
+
```

Comment:
Super nitpicky, but there are a couple blank lines here :) Rubocop would catch that for you.